shared-module/gifio/OnDiskGif.h

```diff
@@ -34,14 +34,20 @@
 
 #include "lib/AnimatedGIF/AnimatedGIF_circuitpy.h"
 #include "shared-module/displayio/Bitmap.h"
+#include "shared-module/displayio/Palette.h"
 
 #include "extmod/vfs_fat.h"
 
+typedef struct {
+    displayio_bitmap_t *bitmap;
+    displayio_palette_t *palette;
+} gifio_ondiskgif_displayio_objs_t;
+
 typedef struct {
     mp_obj_base_t base;
     GIFIMAGE gif;
     pyb_file_obj_t *file;
-    displayio_bitmap_t *bitmap;
+    gifio_ondiskgif_displayio_objs_t displayio_objs;
```

Comment:
Why make this sub-struct?

Reply:
I need to pass the Bitmap and Palette object to the Draw function. Thinking about it now I could just pass the whole `gifio_ondiskgif_t` struct and make it easier to read.